src/init.cpp

```diff
@@ -683,9 +683,8 @@ static void CleanupBlockRevFiles()
 #if HAVE_SYSTEM
 static void StartupNotify(const ArgsManager& args)
 {
-    std::string cmd = args.GetArg("-startupnotify", "");
-    if (!cmd.empty()) {
-        std::thread t(runCommand, cmd);
+    for (std::string command : gArgs.GetArgs("-startupnotify")) {
```

Comment:
```suggestion
    for (std::string cmd : args.GetArgs("-startupnotify")) {
```

Seems odd to use gArgs when args is passed in. Also, why change the symbol name to increase the diff?

Reply:
gArgs: Fixed.

Symbol name: It doesn't increase the diff to adjust it to our current style.